### PR TITLE
fix(table-row): hover and select styling

### PR DESCRIPTION
**Pull request by denkristoffer**
## PR Checklist

- [x] I have read the relevant `readme.md` file(s)
- [x] All commits follow our [Git commit message convention](https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components#commits)
- [x] Tests are added/updated/not required
- [x] Tests are passing
- [x] Storybook stories are added/updated/not required
- [x] Usage notes are added/updated/not required
- [x] Has been tested based on [Contentful's browser support](https://www.contentful.com/faq/about-contentful/#which-browsers-does-contentful-support)
- [x] Doesn't contain any sensitive information

### CODE REVIEW

File: packages/forma-36-react-components/src/components/Table/TableRow/TableRow.css [hunk 1]
```diff
@@ -1,6 +1,10 @@
 @import 'settings/colors';
 
 tbody .TableRow:hover {
+  background-color: var(--color-element-lightest);
+}
+
+.TableRow--selected {
   background-color: var(--color-blue-lightest);
   box-shadow: inset -2px 0 0 var(--color-blue-mid);
```
Comment (mshaaban0) [inline packages/forma-36-react-components/src/components/Table/TableRow/TableRow.css:9]: box shadow is not working on Safari for some reason.

Screenshot 2021-01-14 at 10 29 48

Comment (denkristoffer) [reply packages/forma-36-react-components/src/components/Table/TableRow/TableRow.css:9]: I know, Webkit doesn't support `box-shadow` on `<tr />`.  I tried with a pseudo element instead but that is against the spec and doesn't work at all in any browser. I haven't been able to think of any alternatives.

Comment (suevalov) [reply packages/forma-36-react-components/src/components/Table/TableRow/TableRow.css:9]: I think it's fine to have this regression in Safari.

**Review by suevalov — APPROVED** [review]

File: packages/forma-36-react-components/src/components/Table/TableRow/TableRow.css (hunk 1 above)
Comment (mshaaban0) [reply packages/forma-36-react-components/src/components/Table/TableRow/TableRow.css:9]: Oh wow,TIL. Thanks 🙏

**Review by mshaaban0 — APPROVED** [review]

**Pull request merged**